api/src/pcapi/routes/backoffice/templates/admin/components.html

```diff
@@ -0,0 +1,290 @@
+{% extends "layouts/connected.html" %}
+{% from "components/links.html" import  build_connect_as_pseudo_link with context %}
+{% block page %}
+  <div class="pt-3 px-5">
+    <h2 class="fw-light">Liste des composants du backoffice</h2>
+    <h3 class="fw-light">Actions</h3>
+    <table class="table">
+      <tr>
+        <td>Bouton principal</td>
+        <td>
+          <button class="btn btn-primary">Bouton</button>
+        </td>
+      </tr>
+      <tr>
+        <td>Bouton inversé</td>
+        <td>
+          <button class="btn btn-md btn-outline-primary fw-bold">Bouton</button>
+        </td>
+      </tr>
+      <tr>
+        <td>Bouton inversé secondaire</td>
+        <td>
+          <button class="btn btn-md btn-outline-secondary fw-bold">Bouton</button>
+        </td>
+      </tr>
+      <tr>
+        <td>Boutton désactivé</td>
+        <td>
+          <button class="btn lead fw-bold mt-2"
+                  type="button"
+                  disabled="">Bouton</button>
+        </td>
+      </tr>
+      <tr>
+        <td>Éditer</td>
+        <td>
+          <a href=""><i class="bi-pencil"></i></a>
+        </td>
+      </tr>
+      <tr>
+        <td>Supprimer</td>
+        <td>
+          <a href=""><i class="bi bi-trash3-fill"></i></a>
+        </td>
+      </tr>
+      <tr>
+        <td>Télécharger un CSV</td>
+        <td>
+          <a href=""><i class="bi bi-filetype-csv"></i></a>
+        </td>
+      </tr>
+      <tr>
+        <td>Télécharger un XLSX</td>
+        <td>
+          <a href=""><i class="bi bi-filetype-xlsx"></i></a>
+        </td>
+      </tr>
+      <tr>
+        <td>Voir les autres offres du même lieu</td>
+        <td>
+          <a href=""><i class="bi bi-eye h5"></i></a>
+        </td>
+      </tr>
+      <tr>
+        <td>copier</td>
```

Comment:
Détail : essaye d'harmoniser, avec ou sans majuscule (ici et ensuite)
```suggestion
        <td>Copier</td>
```